src/smart-components/ChannelList/context/ChannelListProvider.tsx

```diff
@@ -203,7 +202,7 @@ const ChannelListProvider: React.FC<ChannelListProviderProps> = (props: ChannelL
   }, [sdkIntialized]);
 
   useEffect(() => {
-    setSdkChannelHandlerId(uuidv4);
+    const sdkChannelHandlerId = uuidv4();
     if (sdkIntialized) {
```

Comment:
This commit shouldnt be here, no?

Reply:
main branch is already updated with this fix, I don't know why GitHub shows like this